src/react-apps/applications/ux-editor/src/types/global.ts

```diff
@@ -65,6 +65,7 @@ declare global {
     textResourceId?: string;
     customType?: string;
     codeListId?: string;
+    dataModelBindings?: IDataModelBindings;
```

Comment:
I like this solution - it's an implementation that actually doesn't break the existing services 👍 

I do wonder if it would be cleaner to keep data model binding consistent for all components, both simple (only one binding) and advanced (multiple bindings). For example by using a standard key (`dataModelBinding`?) for all simple components, and custom keys for advanced components like you have done here. This will of course break all existing services, so we need to decide by the end of this week, @jezpoz @TheTechArch any input on this?

Reply:
Yes, I agree that consistent data model bindings would be cleaner. This appears to be pretty straightforward to implement as well.